packages/forth/0.7.3/build.sh

```diff
@@ -0,0 +1,12 @@
+curl -L https://ftp.gnu.org/gnu/gforth/gforth-0.7.3.tar.gz -o forth.tar.gz
+tar xzf forth.tar.gz
+rm forth.tar.gz
+
+cd gforth-0.7.3/
+./BUILD-FROM-SCRATCH --host=arm --build=arm 
```

Comment:
Any reason we are building for arm vs x86?

Reply:
ok i changed it to x86